src/lib/components/MobileMenu.svelte

```
@@ -281,8 +284,8 @@
 				>
 			</form>
 			<h3 class="text-lg font-bold">Rationale</h3>
-			{#if hasRationale}
-				<div class="py-4 tiptap-content">{@html currentlySelected.explanation}</div>
+			{#if canShowRationale}
+				<div class="py-4 tiptap-content">{@html getRationale(currentlySelected)}</div>
```

ESLint

[error] `{`@html`}` can lead to XSS attack.

(svelte/no-at-html-tags)